### PR TITLE
Enhance the readability

### PR DESCRIPTION
**Pull request by mkumatag**
Changes made:
- Enhanced the readability by adding tables and different sections
- Removed the repeated information.

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/infra/pull/22#" title="Author self-approved">mkumatag</a>*

The full list of commands accepted by this bot can be found [here](?repo=ocp-power-automation%2Finfra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/infra/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: scripts/images/README.md
Comment (bpradipt) [inline scripts/images/README.md:76]: Can we also call out the command to be used? Or name it as CoreOS (RedHat CoreOS)? 
Since for RHCOS the distro we use is `coreos` and not RHCOS.

**Comment by mkumatag** [comment]
@bpradipt Made the changes ptal

File: scripts/images/README.md
Comment (bpradipt) [inline scripts/images/README.md:65]: We should call out that RedHat subscription is only needed for RHEL and not for CentOS

Comment (mkumatag) [reply scripts/images/README.md]: fixed!

**Review by bpradipt — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/infra/pull/22#pullrequestreview-508268763" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/infra/pull/22#" title="Author self-approved">mkumatag</a>*

The full list of commands accepted by this bot can be found [here](?repo=ocp-power-automation%2Finfra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/infra/blob/master/OWNERS)~~ [bpradipt,mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
